testing/test_pytest_html.py

```diff
@@ -1010,7 +1010,15 @@ def test_css_invalid(self, testdir, recwarn):
         result = testdir.runpytest("--html", "report.html", "--css", "style.css")
         assert result.ret
         assert len(recwarn) == 0
-        assert "No such file or directory: 'style.css'" in result.stderr.str()
+        assert "Missing CSS file: style.css" in result.stderr.str()
+
+    def test_multiple_css_files(self, testdir, recwarn):
```

Comment:
Can you add a test for where only part of the passed files are missing? You can use https://github.com/pytest-dev/pytest-html/blob/master/testing/test_pytest_html.py#L321 as an example for creating adhoc files in tests.

Also, please use [@pytest.mark.parametrize](https://docs.pytest.org/en/stable/parametrize.html) to combine your tests with the test above. It'll be a lot cleaner than having 2-3 separate tests for the same feature ( reporting of missing css files )

[Here](https://github.com/pytest-dev/pytest-html/blob/master/testing/test_pytest_html.py#L482) is an example of how we parametrize in this file

Reply:
@gnikonorov See https://github.com/pytest-dev/pytest-html/pull/390/commits/8ccc1045cfa80b463994feb5f418f96a613bcfd9
Have I used `@pytest.mark.parametrize` correctly ?

Reply:
Almost @prakhargurunani! Please see my last review